app/model_config.py

```diff
@@ -0,0 +1,173 @@
+import os
+
+# This is directory tree structure for models and their metadata inside of
+# Google Cloud Storage. The top-level item (e.g. "bert_classification_models")
+# is called the 'bucket'.
+# .
+# └── bert_classification_models
+#     └── multilabel
+#         └── UPR_2percent_ps0
+#             └── saved_model
+#                 ├── label.vocab
+#                 └── 1573031002
+#                     ├── saved_model.pb
+#                     └── variables
+#                         ├── variables.data-00000-of-00001
+#                         └── variables.index
+#         └── UPR_2percent_ps512
+#             └── saved_model
+#                 ├── label.vocab
+#                 └── 1572866595
+#                     ├── saved_model.pb
+#                     └── variables
+#                         ├── variables.data-00000-of-00001
+#                         └── variables.index
+
+
+IN = {
+    "bucket_name": "bert_classification_models",
+    "model_name": "UPR_2percent_ps0",
+    "instance_name": "1573031002",
+    "saved_model": {
+        "name": "saved_model.pb",
+        "path": "multilabel/{model}/saved_model/{instance}",
+    },
+    "variables": {
+        "name": "*",
+        # TODO: come up with a convention on handling recursive copying in the
+        # input configuration.
+        "path": "multilabel/{model}/saved_model/{instance}/variables",
+    },
+    "vocab": {
+        "name": "label.vocab",
+        "path": "multilabel/{model}/saved_model",
+    },
+}
+
+
+class ModelConfig(object):
+    def __init__(self, conf_dict):
+        self._config = conf_dict
+
+    def get_property(self, property_name: str):
+        if property_name not in self._config.keys():
+            return None
+        return self._config[property_name]
+
+
+class PathConfig(ModelConfig):
+
```

Comment:
shouldn't format remove this line?

Reply:
Added a docstring instead. And I think a newline before a method is fine: https://www.python.org/dev/peps/pep-0008/#id21